api/register/swagger-api_register.yaml

```diff
@@ -39,6 +39,36 @@ paths:
           description: Invalid information provided
         403:
           description: Invalid RECAPTCHA token
+  /register/realms/{realm}/user:
```

Comment:
I cannot put my comment on /register/user as nothing as changede but as we now have two modes for registration maybe we should describe in the initial version that user is created in trustID realm and realm parameter is used to determine redirection to use at the end of registration process.